forem/forem
# Update `device_detector`, remove `warning` workaround

## What type of PR is this? (check all applicable)


## Description

Now that the upstream `device_detector` project fixed their regular expressions, the gem also released a new version with these fixes. This means we can upgrade and no longer need the `warning` gem and its intializer.

Note: I briefly considered not removing the `warning` gem, but I guess now that we know it exists we can always bring it back later if needed.


## QA Instructions, Screenshots, Recordings

Start `rails console`, notice that there are no warnings about regular expressions. 

## Added/updated tests?

- [X] No, and this is why: no related tests

## [Forem core team only] How will this change be communicated?
